sri/templatetags/sri.py

```diff
@@ -8,28 +8,42 @@
 from django.utils.safestring import mark_safe
 
 from sri.algorithm import DEFAULT_ALGORITHM, Algorithm
+from sri.extra_attribute import ExtraAttribute
 from sri.integrity import calculate_integrity_of_static
 
 USE_SRI = getattr(settings, "USE_SRI", not settings.DEBUG)
 
 register = template.Library()
 
 
-def sri_js(attrs: dict, path: str, algorithm: Algorithm):
-    attrs.update({"type": "text/javascript", "src": static(path)})
+def sri_js(attrs: dict, path: str, algorithm: Algorithm, **kwargs):
+    if 'extra_attr' in kwargs:
+        extra_attr = kwargs['extra_attr']
+        if ExtraAttribute(extra_attr) in (ExtraAttribute.PRELOAD, ExtraAttribute.PREFETCH):
+            attrs.update({"rel": extra_attr, "href": static(path), "as": "script"})
+        elif ExtraAttribute(extra_attr) in (ExtraAttribute.DEFER, ExtraAttribute.ASYNC):
+            attrs.update({extra_attr: True})
```

Comment:
How does `True` get displayed in the resulting HTML? Really we want these to be just `async defer` rather than `async="true"` etc. Is that already handled by `flatatt`?

Reply:
From `flatatt` help : "In the case of a boolean value, the key will appear without a value".